packages/material-ui/src/IconButton/IconButton.test.js

```diff
@@ -102,6 +102,19 @@ describe('<IconButton />', () => {
     });
   });
 
+  describe('prop: align', () => {
+    it('should render the right class', () => {
```

Comment:
Please split this up into 3 separate test cases. 

The issue is that with this wording it's very hard to determine what broke if that test case is failing i.e. what are the "right classes".  This is mainly an issue if tests are brittle and break when implementation changes even though the user-observable behavior is still correct. It may not seem obvious how a failing test would be a false positive but if that happens it will be very hard to determine what is correct: the test description or the code.

Reply:
Makes sense, will do